stdlib/public/SwiftShims/GlobalObjects.h

```diff
@@ -39,6 +39,8 @@ extern struct _SwiftEmptyArrayStorage _swiftEmptyArrayStorage;
 
 extern __swift_uint64_t _swift_stdlib_HashingDetail_fixedSeedOverride;
 
+extern void *_swift_stdlib_ProcessArguments;
```

Comment:
{{id _Nullable}} gets brought into Swift as {{AnyObject?}}. The only place this is used it's casted to the right kind of pointer immediately. It would be safer too if anyone tried to use it elsewhere. (On the other hand, I guess you're not *supposed* to use it elsewhere.)